src/dashboard/cd-dashboard-stats.vue

```diff
@@ -42,12 +45,15 @@
         dojos: null,
         dojoId: null,
         dojoUsers: null,
-        genders: [],
-        bookedChildren: 0,
+        genders: null,
+        bookedChildren: null,
       };
     },
     computed: {
       ...mapGetters(['loggedInUser']),
+      numberStatText() {
+        return this.$t('{nbKids} kids attended your events', { nbKids: `<span class="cd-dashboard-stats__chart-number-value">${this.bookedChildren}</span>` });
```

Comment:
I know this is the exact same issue as in #198 that I already reviewed, but just to make sure we don't accidentally revert to having `{nbKids}` again.